ansible/roles/kepler_systemd/tasks/validate.yml

```diff
@@ -11,6 +11,7 @@
     done
     echo "Kepler did not start in time"
     sudo systemctl status container-kepler || true
+    sudo podman logs kepler || true
```

Comment:
good addition. I have detected for model validation two errors appearing when logging kepler: "Nov 19 00:08:00 54873a74.packethost.net systemd[1]: Started Podman container-kepler.service.",
        "Nov 19 00:08:00 54873a74.packethost.net podman[53668]: a7af9223880f60c97960b3a284f614c19b5fb05202e06125ab554eab8bea4a9d",
        "Nov 19 00:08:00 54873a74.packethost.net podman[53668]: 2024-11-19 00:08:00.793002311 -0500 EST m=+0.014523666 image pull 5236f0ca9a17d1720c4e046e58f2eeff37249f033bde70ad1ae6514d1639f36b quay.io/sustainable_computing_io/kepler:latest",
        "Nov 19 00:08:00 54873a74.packethost.net kepler[53688]: WARNING: failed to read int from file: open /sys/devices/system/cpu/cpu0/online: no such file or directory",
        "Nov 19 00:08:00 54873a74.packethost.net kepler[53688]: F1119 00:08:00.863081   53690 exporter.go:106] Failed to initialize config: config-dir /etc/kepler/kepler.config does not exist",
        "Nov 19 00:08:00 54873a74.packethost.net podman[53701]: 2024-11-19 00:08:00.881724175 -0500 EST m=+0.013504927 container died. The most notable is the last one which indicates the need for a volume host now.